Appraisals

```diff
@@ -1,4 +1,4 @@
-%w(3.2 4.0 4.1 4.2).each do |version|
+%w(3.2 4.2 5.0).each do |version|
```

Comment:
https://github.com/toptal/chewy/pull/420/files#diff-354f30a63fb0907d4ad57269548329e3R28

They are in martig/include section.